NuEVI/NuEVI.ino

```diff
@@ -280,8 +282,8 @@ void setup() {
     writeSetting(EXTRAC_MAX_ADDR,EXTRAC_MAX_FACTORY);
     writeSetting(CTOUCH_THR_ADDR,CTOUCH_THR_FACTORY);
   }
-  
-  if ((readSetting(VERSION_ADDR) != VERSION) || (!digitalRead(ePin) && !digitalRead(mPin))) {
```

Comment:
This if statement was never entered if version was < 24, since the new version was written to EEPROM in that case.